98_exercises.Rmd

```diff
@@ -291,24 +291,87 @@ See the OMA book for examples.
 
 ### MultiAssayExperiment (MAE) data container
 
-1. Create TreeSE data containers from individual CSV files.
-2. Combine TreeSE into MAE.
-3. Check that each individual experiment of MAE equals corresponding TreeSE.
-4. Take a subset of MAE (e.g., 10 first samples), and observe the subsetted MAE.
-
-Useful functions: DataFrame, TreeSummarizedExperiment, matrix, rownames, colnames, MultiAssayExperiment, ExperimentList, SimpleList
-
+The following exercise walks you through disassembling a MAE object in
+order to store it as multiple separate csv files.
+
+1. Import the microbiomeDataSets package
+2. Store the HintikkaXOData MAE into a variable named `mae`
+3. Which and how many experiments are contained by the MAE?
+4. Split the MAE into TreeSEs named `microbiota`, `metabolites` and `biomarkers`
+5. Which and how many assays are contained by each TreeSE?
```

Comment:
Enough to ask just for one experiment?

Experiments could in principle have other format than TreeSE. Therefore I would call them "experiments" rather than TreeSE's.

Reply:
Pedagogically thinking I think it might be easier to teach them first separately how to create TreeSE object from scractch. Then, MAE construction could be based on just 2-3 ready made TreeSE objects.

How about adding example on how to do this kind of thing with a single TreeSE (in some earlier exercise chapter)? And then here just suggesting to do that exercise for the individual experiments of the MAE, and then in this example focusing exclusively on how MAE is constructed from ready-made TreeSEs? 

I like the idea but wondering if this is getting too broad task at once?